client/src/components/ButtonComponent.jsx

```diff
@@ -30,20 +30,35 @@ const TextStyle = styled.span`
   line-height: 3.6rem;
   `;
 
-class ButtonContainer extends React.Component {
+class ButtonComponent extends React.Component {
   constructor(props) {
```

Comment:
This is a totally valid way of getting rid of your button on click, but it could also be conditionally rendered based off a props variable. Since you are already keeping the onClick at a higher state location, you could use that same function to change the clicked variable(also at the higher state) and pass it in as props to a stateless component, saving you a bunch of boilerplate here. I'm not sure if there is a trade-off to that way of doing things, or if you are planning for this component to utilize its state for other things. Just food for thought